drizzlepac/haputils/pipeline_graphics.py

```diff
@@ -0,0 +1,411 @@
+
+    """
+    # Check for optional elements
+    x_label = plot_dict.get('x_label', '')
+    y_label = plot_dict.get('y_label', '')
+    title = plot_dict.get('title', '')
+    color = plot_dict.get('color', 'blue')
+
+    # Define a figure object with square aspect ratio
+    p1 = HAPFigure(title=title,
+                   x_label=x_label,
+                   y_label=y_label,
+                   use_hover_tips=False)
+
+    # Add the glyphs
+    p1.build_vector_glyph(sourceCDS,
+                          color=color)
+
+    return p1
+
+
+def get_pandas_data(pandas_filename):
+    """Load the harvested data, stored in a CSV file, into local arrays.
+
+    Parameters
+    ==========
+    pandas_filename: str
+        Name of the CSV file created by the harvester.
+
+    Returns
+    =======
+    phot_data: Pandas dataframe
+        Dataframe which is a subset of the input Pandas dataframe written out as
+        a CSV file.  The subset dataframe consists of only the requested columns
+        and rows where all of the requested columns did not contain NaNs.
+
+    """
+    
+    # Instantiate a Pandas Dataframe Reader (lazy instantiation)
+    # df_handle = PandasDFReader_CSV("svm_qa_dataframe.csv")
+    df_handle = PandasDFReader(pandas_filename, log_level=logutil.logging.NOTSET)
+
+    # In this particular case, the names of the desired columns do not
+    # have to be further manipulated, for example, to add dataset specific
+    # names.
+    # 
+    # Get the relevant column data, eliminating all rows which have NaNs
+    # in any of the relevant columns.
+    if pandas_filename.endswith('.h5'):
+        fit_data = df_handle.get_columns_HDF5(HOVER_COLUMNS + RESULTS_COLUMNS)
+        resids_data = df_handle.get_columns_HDF5(RESIDS_COLUMNS)
+    else:
+        fit_data = df_handle.get_columns_CSV(HOVER_COLUMNS + RESULTS_COLUMNS)
+        resids_data = df_handle.get_columns_CSV(RESIDS_COLUMNS)
+
+    return fit_data, resids_data
+
+
+
+def generate_summary_plots(fitCDS, output='cal_qa_results.html'):
+    """Generate the graphics associated with this particular type of data.
+
+    Parameters
+    ==========
+    fitCDS : ColumnDataSource object
+        Dataframe consisting of the relative alignment astrometric fit results
+    
+    output : str, optional
+        Filename for output file with generated plot
+         
+    Returns
+    -------
+    output : str
+        Name of HTML file where the plot was saved.
+
+    """
+    # TODO: include the date from the input data as part of the html filename
+    # Set the output file immediately as advised by Bokeh.
+    output_file(output)
+
+    num_of_datasets = len(fitCDS.data['index'])
+    print('Number of datasets: {}'.format(num_of_datasets))
+        
+    plot_list = []
+
+    # Data point figures
+    p1 = HAPFigure(title='RMS Values',
+                   x_label="RMS_X (pixels)",
+                   y_label="RMS_Y (pixels)",
+                   tools=FIGURE_TOOLS)
```

Comment:
the "tools=FIGURE_TOOLS" should be removed from the parameter list in all places in this code.  You will get the default set of tools defined in graph_utils.py.

Reply:
Removed (in all locations) in 057ba8a.